### DISCUSSION TITLE
Hot reload is not working. I don't know how to troubleshoot it?

### DISCUSSION THREAD

**Discussion by kovkev** [webpack/webpack#15849] [Q&A]
I made a simple webpage like `<Text>{"9"}</Text>` . I then changed it to `<Text>{"10"}</Text>`. This is what happens. Clearly, the client did receive a 10 somewhere, but it did not install it on the webpage successfully:

![image](https://user-images.githubusercontent.com/30991911/169446118-17b62a03-5085-4955-892e-b6a1738b2310.png)

**Comment by vankop** [comment]
please create small reproducible repo
